py/circuitpy_mpconfig.mk

```diff
@@ -26,6 +26,18 @@
 # Boards default to all modules enabled (with exceptions)
 # Manually disable by overriding in #mpconfigboard.mk
 
+enable-if-any=$(lastword $(sort $(1) 0))
+enable-if-all=$(firstword $(sort $(1) 1))
```

Comment:
I wondering if a more explicit name would be good: `is-1-if-all-1`, `is-1-if-any-1`, or maybe you can think of a better name.